core/Cargo.toml

```diff
@@ -21,8 +21,8 @@ futures-timer = "3"
 instant = "0.1.11"
 libsecp256k1 = { version = "0.7.0", optional = true }
 log = "0.4"
-multiaddr = { version = "0.16.0" }
-multihash = { version = "0.16", default-features = false, features = ["std", "multihash-impl", "identity", "sha2"] }
+multiaddr = { version = "0.17.0" }
```

Comment:
`multiaddr` is reexported and exposed on the public `libp2p-core` interface. Thus this is a breaking change and thereby requires version bumps across most of our crates. I can push a commit that does that in a bit.